crates/common/src/zk_compile.rs

```diff
@@ -189,19 +189,20 @@ pub fn compile_smart_contracts(
     is_legacy: bool,
     zksolc_manager: ZkSolcManager,
     project: Project,
-) -> eyre::Result<()> {
+) -> eyre::Result<(ProjectCompileOutput, ContractBytecodes)> {
     let zksolc_opts = ZkSolcOpts {
         compiler_path: zksolc_manager.get_full_compiler_path(),
         is_system,
         force_evmla: is_legacy,
     };
 
     let mut zksolc = ZkSolc::new(zksolc_opts, project);
+    let compile_result = zksolc.compile();
 
-    match zksolc.compile() {
-        Ok(_) => {
-            info!("Compiled Successfully");
-            Ok(())
+    match compile_result {
+        Ok(output) => {
+            println!("Compiled Successfully");
+            Ok(output)
```

Comment:
We need to return the `ProjectCompileOutput` as it's used in `get_project_and_output`